conversant/prompt_chatbot.py

```diff
@@ -498,15 +498,6 @@ def configure_client(self, client_config: Dict = {}) -> None:
             }
         # Override default config values with the config passed in
         if isinstance(client_config, Dict):
-            if "model" in client_config:
```

Comment:
This is so that we're able to use personas where there are no stop sequences at all, without having this injection behavior by default.